x-pack/plugins/apm/server/routes/services.ts

```diff
@@ -91,10 +95,77 @@ export const serviceAnnotationsRoute = createRoute(() => ({
     const { serviceName } = context.params.path;
     const { environment } = context.params.query;
 
+    let annotationsClient: ScopedAnnotationsClient | undefined;
```

Comment:
Can't you do it like this?
```suggestion
    const annotationsClient = await context.plugins.observability?.getScopedAnnotationsClient(
      request
    );
```

Reply:
Changed, thanks!